dissect/target/plugins/os/windows/tasks.py

```diff
@@ -71,6 +76,89 @@
     ],
 )
 
+SchedLgURecord = TargetRecordDescriptor(
+    "windows/tasks/log/schedlgu",
+    [
+        ("datetime", "ts"),
+        ("string", "job"),
+        ("string", "command"),
+        ("string", "status"),
+        ("uint32", "exit_code"),
+        ("string", "version"),
+    ],
+)
+
+
+@dataclass(order=True)
+class SchedLgU:
+    ts: datetime = None
+    job: str = None
+    status: str = None
+    command: str = None
+    exit_code: int = None
+    version: str = None
+
+    @staticmethod
+    def _sanitize_ts(ts: str) -> datetime:
+        # sometimes "at" exists before the timestamp
+        ts = ts.strip("at ")
+
+        try:
+            ts = datetime.strptime(ts, "%m/%d/%Y %I:%M:%S %p")
+        except ValueError:
+            ts = datetime.strptime(ts, "%d-%m-%Y %H:%M:%S")
+
+        return ts
+
+    @staticmethod
+    def _parse_job(line: str) -> tuple[str, str]:
+        job, command = line.split("(", maxsplit=1)
+        command = command.rstrip(")")
+        job = job.strip('"').rstrip('" ')
+
+        return job, command
```

Comment:
Maybe do something like this? (but make the pattern again a global attribute of the class and maybe do some validation for the match ;)
```suggestion
        pattern = re.compile(r"\"([\w\s.]*)\" \(([\w\s.]*)\)")
        match = re.match(pattern, line)
        
        return match.group(1), match.group(2)

        return job, command
```